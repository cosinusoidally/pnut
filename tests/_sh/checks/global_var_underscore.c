// expect_comp_failure
int _IFS;

void main() {
  return;
}
