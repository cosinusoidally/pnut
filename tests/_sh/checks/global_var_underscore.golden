_IFS tests/_sh/checks/global_var_underscore.c:3:0  variable name is invalid. It can't start or end with '_'.
